styles.css

```diff
@@ -0,0 +1,96 @@
+:root {
+  --color-primary: #6070ff;
+  --color-primary-light: #7f8cff;
+  --color-text: #344563;
+  --color-text-dark: #172b4d;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 50%;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-size: 1.6rem;
+  font-weight: 400;
+  font-style: normal;
+  min-height: 100vh;
+  line-height: 2.4rem;
+  background-color: #fff;
+  color: var(--color-text);
+  text-align: left;
+}
+
+.nav_container {
+  height: 5rem;
+  color: var(--color-primary);
+  font-weight: 700;
+  font-size: 1.8rem;
+  line-height: 2rem;
+  padding: 2rem;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav_container > h3 {
+  display: inline-block;
+}
+
+.nav_container > *:hover {
+  cursor: pointer;
+}
+
+.header {
+  background-image: url("assets/header-shapes mobile.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 2rem;
+}
```

Comment:
- [ ] You're doing great so far in your design. It would be nice you fix your styles to make your deadline design look like the Figma design
- [ ] Figma Design
![image](https://user-images.githubusercontent.com/50941074/158655864-d1e59195-ae52-4b79-92b7-9340e25a4d2b.png)

- [ ] Your Design
![image](https://user-images.githubusercontent.com/50941074/158655406-7e1f862f-8b77-4d5d-80d5-841790219a7b.png)